plugins/woocommerce/tests/api-core-tests/tests/data/data-crud.test.js

```diff
@@ -3059,6 +3071,18 @@ test.describe('Data API tests', () => {
 							"thousand_sep": ".",
 							"weight_unit": "kg",
 							"states": expect.arrayContaining([])
+						},
+						{
+							"code": "XK",
+							"name": "Euro",
+							"currency_code": "EUR",
+							"currency_pos": "right_space",
+							"decimal_sep": ",",
+							"dimension_unit": "cm",
+							"num_decimals": 2,
+							"thousand_sep": " ",
+							"weight_unit": "kg",
+							"states": []
```

Comment:
I think we unintentionally added the Kosovo data here alongside the data for South America.

Reply:
I keep losing track scrolling up and down, I should find a better way to manage that. Hopefully I added it in the right continent now.

Reply:
Yeah, I think working with large JSON blobs is generally difficult. 

That's why we use [snapshot testing](https://jestjs.io/docs/snapshot-testing) in Calypso and Gutenberg - it's much easier to generate the changes automatically and review them, rather than to have to generate the fixture yourself.